reference_simulations/second_generation/catalog_files/gleam.vot

```diff
@@ -0,0 +1 @@
+../../first_generation/catalog_files/gleam.vot
```

Comment:
I don't think we want a symlink tracked in the repo. The recommended practice should be to use the `download_gleam` script in pyradiosky and have it write to the correct path.

Reply:
Ok, I can remove that. But what if we instead get rid of `.../first_generation/catalog_files` and `.../second_generation/catalog_files` and instead put everything in just `.../catalog_files`?  (and then change yaml files to point both generations to the right path).

Reply:
@aelanman what do you think about this solution?

Reply:
That sounds good to me!